interface/patient_file/history/history_sdoh.php

```diff
@@ -265,18 +270,18 @@ function render_list_select($field, $list_id, $current, $placeholder = 'Select..
                         </div>
                         <div class="form-group">
                             <label><?php echo xlt("Interventions / Referrals"); ?></label>
-                            <textarea class="form-control" rows="3" name="interventions"><?php echo text(v($info, 'interventions')); ?></textarea>
+                            <textarea class="form-control" rows="3" name="interventions" placeholder="<?php echo xla('Assessment calculated Treatment Plan Interventions resources display here after assessment is saved.') ?>" readonly><?php echo text($interventions_text); ?></textarea>
                         </div>
                     </div>
                 </div>
 
                 <div class="mb-4">
                     <button type="submit" class="btn btn-primary"><?php echo xlt("Save"); ?></button>
-                    <a class="btn btn-secondary" href="<?php echo attr($GLOBALS['webroot'] . '/interface/patient_file/history/history_sdoh_widget.php?pid=' . urlencode($pid)); ?>">
```

Comment:
i also usually don't escape the entire url and could focus it on just the $pid with attr_url (same for line 284 below)